### DISCUSSION TITLE
ArchR LSI combined function not working

### DISCUSSION THREAD

**Discussion by millerkaplanlab** [GreenleafLab/ArchR#1774] [General]
Hello: I am having issues with the following code when trying to combine RNA and ATAC seq data using the multiome ArchR pipeline.

proj <- addCombinedDims(proj, reducedDims = c("LSI_ATAC", "LSI_RNA"), name =  "LSI_Combined")
Error in f(init, x[[i]]) : 
  number of rows of matrices must match (see arg 2)

**Comment by rcorces** [comment]
Sorry for the delay in my reply. Errors should be posted to Issues and follow the provided Issue Template. Please make sure to indicate whether you are able to recapitulate your error with the tutorial data.
